### ISSUE TITLE
About Slice Layer

### ISSUE THREAD

**Issue by weiyichang** [nerox8664/onnx2keras#35]
Hello,

I have a problem related to slice layer .

My model is like this 
(I want to slice the ch dimension, and then concatenate them, i.e.,
 I set axis attribute as 1 in slice layer in an onnx file)

<img width="198" alt="截圖 2019-12-04 下午3 36 40" src="https://user-images.githubusercontent.com/20596865/70122770-2d64fb00-16ac-11ea-80a6-20f44da913f7.png">

I add a slice layer after the first conv layer
but while converting this model to tflite model, it will show that the input of concat layer  is mismatch

I print the shape of slice layer like this:

![截圖 2019-12-04 下午3 35 10](https://user-images.githubusercontent.com/20596865/70123038-c72ca800-16ac-11ea-871e-17b758f172d4.png)

For the upper part, it is normal (slice input to two tensors with 4 channels and 12 channels), 

but for lower part, it slices the wrong dimension

I use the setting of change_ordering=True with onnx2keras==0.0.17

Do you have any suggestion ? Thanks a lot :)

**Comment by krtbb** [comment]
Hello.

I encountered a same issue, and the cause seems to be in `tf.nn.strided_slice()` according to my test.
I fixed it and you can tell by looking at the PR #55 .

Best regards.
